Framework/Kernel/test/PropertyWithValueJSONTest.h

```diff
@@ -138,13 +138,39 @@ class PropertyWithValueJSONTest : public CxxTest::TestSuite {
   }
 
 private:
-  template <typename ValueType>
-  void doSingleValueObjectDecodeTest(const std::string &propName, const ValueType &propValue) {
+  // This would be templated but there is a bug in GCC that prevents explicit specialization inside of a class. So these
```

Comment:
Is there a gcc bug report for some particular version of gcc? Do we know why `doBasicDecodeTest` works?